Apps/Database/src/Models/UserProfile.cs

```diff
@@ -72,5 +73,10 @@ public class UserProfile : AuditableEntity
         /// </summary>
         [MaxLength(44)]
         public string? EncryptionKey { get; set; }
+
+        /// <summary>
+        /// Gets the list containing all the communication emails related to this user profile.
+        /// </summary>
+        public virtual List<CommunicationEmail> CommunicationEmails { get; } = new List<CommunicationEmail>();
```

Comment:
I don't think we have the other objects on the profile table. Is this necessary to get the foreign key relationship to work?

Reply:
removing it to try now.